src/Console/Commands/ImportUsers.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace Statamic\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Database\Eloquent\Concerns\HasUuids;
+use Illuminate\Support\Facades\Schema;
+use Statamic\Auth\Eloquent\User as EloquentUser;
+use Statamic\Auth\File\User as FileUser;
+use Statamic\Auth\UserRepositoryManager;
+use Statamic\Console\RunsInPlease;
+use Statamic\Contracts\Auth\User as UserContract;
+use Statamic\Contracts\Auth\UserRepository as UserRepositoryContract;
+use Statamic\Facades\User;
+use Statamic\Stache\Repositories\UserRepository as FileRepository;
+
+class ImportUsers extends Command
+{
+    use RunsInPlease;
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'statamic:eloquent:import-users';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Imports file based users into the database.';
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        if (config('statamic.users.repository') === 'eloquent') {
+            $this->importUsers();
+        }
+
+        return 0;
+    }
+
+    private function importUsers()
+    {
+        $guard = config('statamic.users.guards.cp');
+        $provider = config("auth.guards.$guard.provider");
+        $model = config("auth.providers.$provider.model");
+
+        if (! in_array(HasUuids::class, class_uses_recursive($model))) {
+            $this->error('Your user model must use the HasUuids trait for this migration to run');
+
+            return;
+        }
+
+        if (! str_contains(Schema::getColumnType('users', 'id'), 'uuid') && ! str_contains(Schema::getColumnType('users', 'id'), 'string')) {
```

Comment:
Sorry for the delay. I just wanted to know how you got `'uuid'` to appear as a type?

If I modify the initial `create_users_table` migration use uuids, the column type is `string`.

```diff
-$table->id();
+$table->uuid('id');
```

```php
Schema::getColumnType('users', 'id'); // 'string'
```

If I change the column within the `statamic_auth_tables` migration, the column type is `guid`.

```diff
+$table->uuid('id')->change();
```
```php
Schema::getColumnType('users', 'id'); // 'guid'
```

(The different results may or may not be a Laravel issue, but that doesn't matter at the moment.)

I'm curious how you ever got `uuid`?